src/pages/singer/Spec.vue

```diff
@@ -1,238 +1,29 @@
-    "value": 30.01, "tags": {"endpoint": "orders", "http_status_code": 500, "status": "failed"}}
-    This can be interpreted as: an HTTP request to an “orders” endpoint was made that took 30.01
-    seconds and failed with a status code of 500. Counter for Records # INFO METRIC: {"type":
-    "counter", "metric": "record_count", "value": 100, "tags": {"endpoint": "orders"}} INFO METRIC:
-    {"type": "counter", "metric": "record_count", "value": 100, "tags": {"endpoint": "orders"}} INFO
-    METRIC: {"type": "counter", "metric": "record_count", "value": 100, "tags": {"endpoint":
-    "orders"}} INFO METRIC: {"type": "counter", "metric": "record_count", "value": 14, "tags":
-    {"endpoint": "orders"}} This can be interpreted as: a total of 314 records were fetched from an
-    “orders” endpoint. Log Output # Metrics messages are interspersed with the primary 3 messages,
-    so parsing them should be handled programmatically. This is an example of what a realistic log
-    output might look like: INFO Using API Token authentication. INFO tickets: Skipping - not
-    selected {"type": "SCHEMA", "stream": "groups", "schema": {"properties": {"name": {"type":
-    ["string"]}, "created_at": {"format": "date-time", "type": ["string"]}, "url": {"type":
-    ["string"]}, "updated_at": {"format": "date-time", "type": ["string"]}, "deleted": {"type":
-    ["boolean"]}, "id": {"type": ["integer"]}}, "type": ["object"]}, "key_properties": ["id"]} INFO
-    groups: Starting sync INFO METRIC: {"type": "timer", "metric": "http_request_duration", "value":
-    0.6276309490203857, "tags": {"status": "succeeded"}} {"type": "RECORD", "stream": "groups",
-    "record": {"id": 360007960773, "updated_at": "2020-01-09T09:57:16.000000Z"}} {"type": "STATE",
-    "value": {"bookmarks": {"groups": {"updated_at": "2020-01-09T09:57:16Z"}}}} Targets # When
-    targets are run, they can accept a single config file that provides the information necessary
-    for it to work properly. Config Files # Similar to taps, targets take a configuration file.
-    There is no specification for the structure of a config file, as long as it is JSON based. State
-    Files # Unlike taps, targets do not take a state file. Targets are expected to read in the state
-    messages from stdin, but typically they do not do anything with the state messages beyond
-    sending them to stdout. This is done once all data that appeared in the stream before the state
-    message has been processed by the Target. Schema Files # Targets do not take a schema file.
-    However, they are expected to read the schema messages from stdin and perform validation of the
-    incoming record using the provided schema.
+    <div class="grid w-full">
+      <div class="h-7 max-h-7"></div>
+      <div class=" bg-white border text-black w-full">
+        <p class="text-3xl py4">Singer Spec</p>
+        TODO: Add rendered content here from <code>/src/pages/singer/spec_content.md</code>
```

Comment:
- [ ] TODO: @aaronsteers

Reply:
Ditto. Resolved in 3012d832dbbce3d33564479baf9b0e0909fdc3c8 and aee1f4cead0db1de8d7d8d26d01337642889592f.